src/tests/unittests/test_filmicrgb.c

```diff
@@ -0,0 +1,145 @@
+#include <cmocka.h>
+#include <limits.h>
+#include <setjmp.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <stdio.h>
+
+#include "iop/filmicrgb.c"
+
+// epsilon for floating point comparison (TODO: take more sophisticated value):
+#define E 1e-6f
+
+/******************************************************************************
+ * MOCKED FUNCTIONS
+ */
+void __wrap_dt_iop_color_picker_reset(dt_iop_module_t *module, gboolean update)
+{
+  check_expected_ptr(module);
+  check_expected(update);
+}
+
+dt_iop_order_iccprofile_info_t *__wrap_dt_ioppr_get_pipe_work_profile_info(struct dt_dev_pixelpipe_t *pipe)
+{
+  return NULL;
+}
+
+/******************************************************************************
+ * TEST FUNCTIONS
+ */
+static void test_sample(void **state)
+{
+  expect_value(__wrap_dt_iop_color_picker_reset, module, NULL);
+  expect_value(__wrap_dt_iop_color_picker_reset, update, TRUE);
```

Comment:
`will_return(__wrap_dt_ioppr_get_pipe_work_profile_info, NULL);`

Instruct the mock to return NULL ...